.azure/pipelines/helix-matrix.yml

```diff
@@ -1,5 +1,9 @@
 # We only want to run full helix matrix on main
-pr: none
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - '*'
```

Comment:
Don't forget to revert 😺

Reply:
Yep, likely will need to add some SkipHelixQueues from the helix-matrix run, i.e. ubuntu 1604

Reply:
Skipped all the additional queues because each linux distro seems to have different reqs missing, will file an issue to track deciding if we care, might not be worth running the playwright tests everywhere.  The selenium tests were windows only so we are still gaining osx/ubuntu compared to before